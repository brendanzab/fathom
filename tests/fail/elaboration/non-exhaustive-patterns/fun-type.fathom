//~ exit-code = 1

fun (1 : U32) -> U32
