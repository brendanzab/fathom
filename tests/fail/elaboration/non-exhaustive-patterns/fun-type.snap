stdout = ''
stderr = '''
error: non-exhaustive patterns in match expression
  ┌─ tests/fail/elaboration/non-exhaustive-patterns/fun-type.fathom:3:6
  │
3 │ fun (1 : U32) -> U32
  │      ^ patterns not covered

'''
